index.html

```diff
@@ -1,66 +1,354 @@
+            <ul id="cardElements">
+              <li>
+                <h1 id="cardTitle">Profesional Art Printing Data</h1>
+              </li>
+              <li>
+                <h1 id="cardDesc">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</h1>
+              </li>
+              <li id="cardTags">
+                <ul class="flex-container" id="tagGroup">
+                  <li class="tag">
+                    <a class="tagText" id="html">html</a>
+                  </li>
+                  <li class="tag">
+                    <a class="tagText" id="bootstrap">bootstrap</a>
+                  </li>
+                  <li class="tag">
+                    <a class="tagText" id="Ruby">Ruby</a>
+                  </li>
+                </ul>
+            </ul>
+            <div id="cardButton">
+              <button class="button" type="button">
+                <span id="buttonText">See Project</span>
+              </button>
+            </div>
+          </div>
+      </div>
+      <div class="flex-container" id="aboutMyself">
+        <ul class="flex-container" id="aboutMe">
           <li>
-            <h1 id="primary-text">Hey There. I'm Eduardo</h1>
+            <h1 id="amTitle">About me</h1>
           </li>
           <li>
-            <h1 id="supporting-text">I'am a Software Developer</h1>
+            <h1 id="amDesc">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</h1>
           </li>
           <li>
-            <h1 id="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</h1>
+            <button class="button" type="button">
+              <span id="buttonText">Get My Resume</span>
+            </button>
           </li>
-          <li>
-            <ul id="social-media">
+        </ul>
+        <ul class="flex-container" id="skills">
+          <li class="flex-container" id="skillBlock">
+            <div class="flex-container" id="skill">
+              <img class="skillImg" id="lanIcon"/ alt="Language">
+              <h1 id="skillTxt">Language</h1>
+            </div>
+            <ul class="flex-containers" id="skillFeatures">
+              <li>
+                <h1 class="featureText">Javascript</h1>
+              </li>
+              <li>
+                <h1 class="featureText">Ruby</h1>
+              </li>
+              <li>
+                <h1 class="featureText">Html</h1>
+              </li>
+              <li>
+                <h1 class="featureText">Css</h1>
+              </li>
+            </ul>
+          </li>
+          <li class="flex-container" id="skillBlock">
+            <div class="flex-container" id="skill">
```

Comment:
The section image-icon of your design is not currently showing, please kindly fix this issue

Your design
![image](https://user-images.githubusercontent.com/78251927/212381785-3f68788b-3480-408b-9ec8-d594328fba02.png)

Figma
![image](https://user-images.githubusercontent.com/78251927/212381886-811d0b18-4f21-451d-919e-c551baef1da5.png)